src/styles/blocks/card.scss

```diff
@@ -0,0 +1,38 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px;
+  box-shadow: 1px 1px 4px gray;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    padding-top: 33px;
+    padding-left: 20px;
+  }
+
+  &__title {
+    display: flex;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 14px;
+    width: 166px;
+    padding-top: 40px;
+    padding-left: 17px;
+  }
```

Comment:
The `&__title` block has fixed width and padding which may not be very flexible. Consider using more responsive units like percentages or `em` for the width and padding to make the layout more adaptable to different screen sizes or content changes.